yarn.lock

```diff
@@ -5,22 +5,19 @@
 "@ant-design/create-react-context@^0.2.4":
   version "0.2.5"
   resolved "https://registry.yarnpkg.com/@ant-design/create-react-context/-/create-react-context-0.2.5.tgz#f5f5a9163b4772097712837397ad30e22e79f858"
-  integrity sha512-1rMAa4qgP2lfl/QBH9i78+Gjxtj9FTMpMyDGZsEBW5Kih72EuUo9958mV8PgpRkh4uwPSQ7vVZWXeyNZXVAFDg==
```

Comment:
@robyww These are the sha hash I was refering to.  I wasn't sure if you had to do anything special to get them.  I did the normal `yarn install` and did not get them.  I don't think it's a big deal though.